README.md

```diff
@@ -5,26 +5,37 @@
 [![codecov](https://codecov.io/gh/City-of-Helsinki/notification-service-api/branch/develop/graph/badge.svg)](https://codecov.io/gh/City-of-Helsinki/notification-service-api)
 
 ## Environments
-Production environment:
-- https://api.hel.fi/notification-service/v1
 
 Testing environment:
-- https://notification-service-api.test.kuva.hel.ninja/v1
+- https://kuva-notification-service.api.test.hel.ninja/v1/
 
 ## Development with Docker
 
 1. Copy `docker-compose.env.yaml.example` to `docker-compose.env.yaml` and modify it if needed. Be sure that database variables are set (DATABASE_URL and DATABASE_HOST) because those were removed from 'docker-compose.yml' file.
 
-2. Run `docker-compose up`
+2. <a name="hostname1">Set keycloak hostname</a>
 
-The project is now running at [localhost:8081](http://localhost:8081)
+   Add the following line to your hosts file (`/etc/hosts` on mac and linux):
+    ```
+    127.0.0.1       notification-service-keycloak
+    ```
+
+3. Run `docker compose up` or `docker compose up --force-recreate --build` if you have made changes, for example,  to environmental variables.
+
+The project is now running at [localhost:8081](http://localhost:8081).
+Keycloak admin interface is running at [notification-service-keycloak:8180/admin](http://notification-service-keycloak:8180/admin).
+
+The Keycloak development realm for this project is preconfigured with the name "locale-dev". The basic configuration is derived from the file `realm.json`, which is imported during the build process. Realm parameters can be modified through environmental variables in `docker-compose.keycloak.env.yaml` and `docker-compose.env.yaml`. It's essential to ensure that parameters match between the configuration files and the running Keycloak instance to enable the default authentication flow properly.
```

Comment:
If it's not a huge task I'd heartily recommend fixing the typo `locale-dev` -> `local-dev` because having locale/language related word in the name is misleading and can be problematic later as it brings up questions what does it mean and why is there something locale/language related in the name IMHO.

Reply:
The point being that:
 - locale = language = kieli
 - local = paikallinen

Reply:
Replaced the occurrences.